sky/cli.py

```diff
@@ -3055,8 +3056,11 @@ def show_gpus(
             '--all-regions and --region flags cannot be used simultaneously.')
 
     # This will validate 'cloud' and raise if not found.
-    cloud_obj = sky_clouds.CLOUD_REGISTRY.from_str(cloud)
-    service_catalog.validate_region_zone(region, None, clouds=cloud)
+    cloud_obj = sky_clouds.CLOUD_REGISTRY.from_name_or_alias(cloud)
+    cloud_name = None
+    if cloud_obj is not None:
+        cloud_name = type(cloud_obj).__name__.lower()
```

Comment:
Might be a little cleaner to have a reverse lookup in `CLOUD_REGISTRY`, e.g., `CLOUD_REGISTRY.to_str(cloud)`?